gradle/spec/dependabot/gradle/update_checker_spec.rb

```diff
@@ -142,6 +142,25 @@
 
       it { is_expected.to eq(version_class.new("23.0")) }
     end
+
+    context "with a git source" do
+      let(:dependency_requirements) do
+        [{
+          requirement: nil,
+          file: "package.json",
```

Comment:
Can you update this so it looks like a Gradle example (just because it's surprising otherwise)?

Reply:
Good catch!  I added a new commit.